### PR TITLE
[1.3.x] chore(deps): bump bulk import frontend plugin

### PR DESCRIPTION
**Pull request by debsmita1**
manual cherry-pick of https://github.com/janus-idp/backstage-showcase/pull/1626

### CODE REVIEW

**Review by rm3l — COMMENTED** [review]
I can close https://github.com/janus-idp/backstage-showcase/pull/1612 as superseded by this PR, but I think you'll also need [`0537a2b` (#1612)](https://github.com/janus-idp/backstage-showcase/pull/1612/commits/0537a2b6cc4ea9a1597d903caac67ba7489506fa) here (it was needed in the [original PR upgrading the bulk import backend plugin](https://github.com/janus-idp/backstage-showcase/pull/1605/files#diff-8d67f14bec3e8fbb7718ffb4a47c2f9534411a556332a1ea531a231edeb3ff08), for the E2E tests to pass).

File: dynamic-plugins/imports/package.json [hunk 1]
```diff
@@ -27,9 +27,9 @@
     "@janus-idp/backstage-plugin-quay": "1.11.3",
     "@janus-idp/backstage-plugin-rbac": "1.29.4",
     "@janus-idp/backstage-plugin-tekton": "3.12.3",
-    "@janus-idp/backstage-plugin-bulk-import": "1.4.6",
-    "@janus-idp/backstage-plugin-bulk-import-backend": "1.4.1",
-    "@janus-idp/backstage-plugin-topology": "1.27.3",
+    "@janus-idp/backstage-plugin-bulk-import": "1.4.9",
+    "@janus-idp/backstage-plugin-bulk-import-backend": "1.5.2",
```
Comment (rm3l) [inline dynamic-plugins/imports/package.json:31]:
```suggestion
    "@janus-idp/backstage-plugin-bulk-import-backend": "1.5.3",
```

A new version of the bulk-import backend plugin has just been released, to fix https://issues.redhat.com/browse/RHIDP-4021 (see https://github.com/janus-idp/backstage-plugins/pull/2178).

https://github.com/janus-idp/backstage-showcase/pull/1635 adds it in the `main` branch.

**Comment by debsmita1** [comment]
updated the backend plugin version

**Comment by ciiay** [comment]
Tested the updated version on a cluster with the PR image, and it works great 👏  `argo-ui` did show up in the Catalog list later, I just didn't wait long enough in the screen recording.

https://github.com/user-attachments/assets/88829f2c-83b8-4ecf-a83e-d74217b7160b

image


/lgtm

**Review by invincibleJai — APPROVED** [review]
/lgtm

**Pull request merged**